core/src/main/java/GameObjects/UrlBuilder.java

```diff
@@ -5,5 +5,5 @@
  */
 
 public interface UrlBuilder {
-  String buildUrl(String amount, String category, String difficulty);
+  String buildUrl();
```

Comment:
buildUrl should now be taking in a GameSettings object, rather than creating a new UrlBuilder for each settings object (Essentially replacing the separate string parameters with one object that holds the strings).